.github/taskIdbyowner.py

```diff
@@ -0,0 +1,36 @@
+import os
+import json
+
+codeowners_file_path = '/workspaces/azure-pipelines-tasks/.github/CODEOWNERS'
```

Comment:
It would be better to not check in paths from your local machine. There are some nuances to it, but `from pathlib import Path` and `Path(__file__).parent / 'CODEOWNERS'` should get you close enough to what you need